pkg/cmd/cli/cmd/extract.go

```diff
@@ -69,6 +69,7 @@ func NewCmdExtract(fullName string, f *clientcmd.Factory, in io.Reader, out, err
 		Run: func(cmd *cobra.Command, args []string) {
 			kcmdutil.CheckErr(options.Complete(f, in, out, cmd, args))
 			kcmdutil.CheckErr(options.Validate())
+			// TODO: move met to kcmdutil
```

Comment:
yeah, I was meant to type "move me", but there are lots of "move met" exist, I thought it was like kind of cipher or something, anyway, will update all this typo :smile: